docs/install_guides/installing-openvino-brew.md

```diff
@@ -0,0 +1,105 @@
+
+## Prerequisites
+
+### System Requirements
+
+@sphinxdirective
+
+Full requirement listing is available on the `System Requirements Page <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html>`_
+
+@endsphinxdirective
+
+### Software Requirements
+
+@sphinxdirective
+
+.. tab:: Linux
+
+  * `Homebrew <https://brew.sh/>`_
+  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+  * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
+  * `Python 3.7 - 3.10, 64-bit <https://www.python.org/downloads/>`_
+
+.. tab:: macOS
+
+  * `Homebrew <https://brew.sh/>`_
+  * `CMake 3.13 or higher <https://cmake.org/download/>`_ (choose "macOS 10.13 or later"). Add `/Applications/CMake.app/Contents/bin` to path (for default installation). 
+  * `Python 3.7 - 3.10 <https://www.python.org/downloads/mac-osx/>`_ (choose 3.7 - 3.10). Install and add it to path.
+  * Apple Xcode Command Line Tools. In the terminal, run `xcode-select --install` from any directory to install it.
+  * (Optional) Apple Xcode IDE (not required for OpenVINO™, but useful for development)
+
+@endsphinxdirective
+
+## Installing OpenVINO Runtime
+
+@sphinxdirective
+
+1. Make sure that you have installed HomeBrew on your system. If not, follow the instructions on `the Homebrew website <https://brew.sh/>`_ to install and configure it.
+
+2. Open a command prompt terminal window, and run the following command to install OpenVINO Runtime:
+
+   .. code-block:: sh
+
+      brew install openvino
+
+
+Congratulations, you've finished the installation!
+
+@endsphinxdirective
+
+
+## (Optional) Installing Additional Components
+
+OpenVINO Development Tools is a set of utilities for working with OpenVINO and OpenVINO models. It provides tools like Model Optimizer, Benchmark Tool, Post-Training Optimization Tool, and Open Model Zoo Downloader. If you install OpenVINO Runtime using archive files, OpenVINO Development Tools must be installed separately.
+
+See the [Install OpenVINO Development Tools](installing-model-dev-tools.md) page for step-by-step installation instructions.
+
+OpenCV is necessary to run demos from Open Model Zoo (OMZ). Some OpenVINO samples can also extend their capabilities when compiled with OpenCV as a dependency. To install OpenCV for OpenVINO, see the [instructions on GitHub](https://github.com/opencv/opencv/wiki/BuildOpenCV4OpenVINO).
+
+## Uninstalling OpenVINO
+
+To uninstall OpenVINO via HomeBrew, use the following command:
+```sh
+brew uninstall openvino
+```
+
+## What's Next?
+
+@sphinxdirective
+
+Now that you've installed OpenVINO Runtime, you're ready to run your own machine learning applications! Learn more about how to integrate a model in OpenVINO applications by trying out the following tutorials.
+
+* Try the `C++ Quick Start Example <openvino_docs_get_started_get_started_demos.html>`_ for step-by-step instructions on building and running a basic image classification C++ application.
```

Comment:
brew's OpenVINO does not contain samples.
So, users have to download samples from github. Maybe we refer to generic C++ instead